assembl/static2/js/app/components/common/attachFileForm.jsx

```diff
@@ -36,12 +36,18 @@ class AttachFileForm extends React.Component<*, AttachFileFormProps, AttachFileF
   };
 
   render() {
+    let isImage;
+    const file = this.state.file;
+    if (file !== null) {
+      isImage = file.type.includes('image');
+    }
+
     return (
-      <div className="attach-file">
+      <div className={isImage ? 'attach-file with-preview' : 'attach-file'}>
```

Comment:
it is a good habit to use `classnames` lib for this kind of stuff

Reply:
Allright, I thought the classnames lib was only pertinent when we had more than 2 classes alternating